Lombiq.Hosting.Tenants.Admin/Readme.md

```diff
@@ -0,0 +1,46 @@
+# Lombiq Hosting Tenants - Admin
+
+## About
+
+With the help of this module, you can log in from the Default tenant admin dashboard to any other tenants as an administrator user.
+
+## Documentation
+
+This module contains two features:
+- Lombiq.Hosting.Tenants.Admin
```

Comment:
Mivel ez egy technical name, így kellene formázni, meg a másik feature-t is, az összes többi helyen is, címekben is.

```suggestion
- `Lombiq.Hosting.Tenants.Admin`
```